### PR TITLE
Add class and module snippets

### PR DESCRIPTION
**Pull request by ProZsolt**

### CODE REVIEW

File: snippets/language-ruby.cson [hunk 1]
```diff
@@ -99,6 +99,9 @@
   'class .. end':
     'prefix': 'cla'
     'body': 'class $1\n\t$0\nend'
+  'class … end':
```
Comment (50Wliu) [inline snippets/language-ruby.cson]: Is there a reason this is an ellipsis and not `...`?

Comment (ProZsolt) [inline snippets/language-ruby.cson]: Not really. I copied from `def … end`.
Should I change this?

Comment (50Wliu) [inline snippets/language-ruby.cson]: Nah, I didn't know the snippets were already using them.

**Comment by 50Wliu** [comment]
I'll merge this in a few days if no one objects.

**Comment by jnunemaker** [comment]
Why not just alter the existing snippets? These are basically the exact same thing, right?

**Comment by ProZsolt** [comment]
Are multiple prefixes supported? If so we can alter the existing ones, but I don't want to remove the short versions if somebody using them.

**Comment by jnunemaker** [comment]
It would only alter in that they would have to hit backspace to do something other than the class name (say << self or something), right? That seems pretty minor. We also have shortcuts for that case already. I'm trying to think of how your version would hurt those currently using the cla shortcut. Perhaps I'm just not thinking through all the scenarios. Feel free to do whatever on this, I was mostly just trying to understand and say that they seem so close we might as well merge as opposed to adding a new one.

**Comment by ProZsolt** [comment]
I think most of the people who knows the shortcut will use 'cla', because it is shorter, but 'class' is more intuitive

**Comment by jnunemaker** [comment]
Oh, I misunderstood. I though you were saying the result of the snippets was different. You mean just what is triggering the snippet. cla is that way due to previous history with textmate and sublime. I would bet cla is and will be far more popular. I don't see any harm in merging this other than possibly causing confusion, like which one should I use? But that seems minor.

**Comment by ProZsolt** [comment]
Fixed

**Comment by 50Wliu** [comment]
There seem to be some conflicts - can you please rebase on top of master?  Thanks.

**Comment by ProZsolt** [comment]
Rebased

**Comment by 50Wliu** [comment]
@jnunemaker Are you :+1: are :-1: on this?

**Comment by jnunemaker** [comment]
I don't really get why to do it, but it doesn't hurt anything other than providing two ways to do the same thing, which could be initially confusing, so don't let me stand in anyone's way. :)

**Pull request closed without merging**